packages/snap-preact-components/src/components/Molecules/FacetPaletteOptions/FacetPaletteOptions.tsx

```diff
@@ -174,6 +182,7 @@ export interface FacetPaletteOptionsProps extends ComponentProps {
 	columns?: number;
 	gapSize?: string;
 	hideIcon?: boolean;
+	facet?: ValueFacet;
```

Comment:
Made `facet` optional here, but not for `values` above it - likely same for other components with similar change.